### ISSUE TITLE
Make buffer time between last modified on disk and last modified on last save configurable

### ISSUE THREAD

**Issue by blink1073** [jupyterlab/jupyterlab#8556]
## Description
https://github.com/jupyter/notebook/pull/3273 made the buffer time configurable using notebook config.  Currently it is hardcoded in [JupyterLab](https://github.com/jupyterlab/jupyterlab/blob/7204b461515890d86cba5a3b51832708ae265e8a/packages/docregistry/src/context.ts#L626).   This value should be configurable using settings.

**Comment by eduntct** [comment]
Is there a way to modify lastModifiedCheckMargin "System Defaults", rather than on a user-by-user basis?  Thanks!

**Comment by krassowski** [comment]
You could use [`overrides.json`](https://jupyterlab.readthedocs.io/en/stable/user/directories.html#overrides-json) to pre-define settings for all users.

**Comment by eduntct** [comment]
> You could use [`overrides.json`](https://jupyterlab.readthedocs.io/en/stable/user/directories.html#overrides-json) to pre-define settings for all users.

Works like a charm, thanks!

For future reference, I did this (notice I use conda):

Edit /opt/conda/envs/${PYTHON_ENVIRONMENT}/share/jupyter/lab/settings/overrides.json to include:
```
{
  "@jupyterlab/docmanager-extension:plugin": {
    "lastModifiedCheckMargin": 5000
  }
}
```
where PYTHON_ENVIRONMENT stores the name of the virtual environment where your running Jupyter is installed.

Followed by:
```
jupyter lab build
```

**Comment by andrii-i** [comment]
Hey everyone! Was this issue fixed by the merged [PR #11153](https://github.com/jupyterlab/jupyterlab/pull/11153) referenced above? If yes, let's close this PR

**Comment by krassowski** [comment]
Yes, it was, thank you @andrii-i - closing
